Src/AutoFakeItEasyUnitTest/FixtureIntegrationTest.cs

```diff
@@ -46,53 +46,51 @@ public void FixtureCanPassValuesToAbstractGenericTypeWithNonDefaultConstructor()
         [Fact]
         public void FixtureCanCreateFakeOfDelegate()
         {
-            // Fixture setup
+            // Arrange
```

Comment:
How embarrassing. It seems I'd copied the "Fixture setup" etc. from the local copy of the repo I'd had before you made  a2e35d801d52f960a7479d020fdf82835c88c731.

Reply:
No worries 😉 Actually, I thought that it was a matter of habit, as the previous 4 phases ugly style has been here for a while 😄